File: frontend/src/app/models/news.ts
export class News {
    title: string;
    postId: number;
    author: string;
    body: string;
    date: string;
    image: string;

    constructor (title: string, postId: number, author: string, body: string, date: string, image: string) {
        this.title = title;
        this.postId = postId;
        this.author = author;
        this.body = body;
        this.date = date;
        this.image = image;
    }
}

File: server/src/main/java/dev/jaczerob/eras/server/toontown/models/news/News.java
package dev.jaczerob.eras.server.toontown.models.news;

import dev.jaczerob.eras.server.utils.ToontownObject;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = false)
public class News extends ToontownObject {
    private String title;
    private int postId;
    private String author;
    private String body;
    private String date;
    private String image;
}
